test/integration/formatting_stack/strategies.clj

```diff
@@ -235,3 +238,18 @@
         (sh "rm" (str rename-destination))
         (assert (-> deletable-file .exists))
         (assert (not (-> rename-destination .exists)))))))
+
+(deftest namespaces-within-refresh-dirs-only
+  (speced/let [^{::speced/spec (rcomp count (partial < 100))}
```

Comment:
Why is this assertion here? It'll cause an unrelated spec failure when formatting stack exceeds 100 tracked source files.

Reply:
In fact the opposite, read it again. Mistaking `<` happens to the best of us ;p

The point is that `all-files` should return a high number of files, it's an assumption/requirement of this test. If it returned say 0-10 files the output would mismatch the expectations, and we could be getting a false negative.

This assertion must be placed here (and not in `all-files` itself) because there's no generic number that will be true for all projects.

Reply:
The rcomp + partial combination caught me off guard I guess 😅. Esp since the order of the arguments matter a lot if used in this case.


Anyway; sgtm!